src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
```

Comment:
You're querying for all 'li' elements in the entire document. This could potentially select undesired elements if the HTML structure of your page changes in the future. It would be better to be more specific in your selector, for example by using a class or id.